src/runtime/node/cluster/index.ts

```diff
@@ -0,0 +1,82 @@
+import { notImplemented } from "src/runtime/_internal/utils";
+import noop from "../../mock/noop";
+import mock from "../../mock/proxy";
+import type cluster from "node:cluster";
+import type { Cluster, Worker as _Worker } from "node:cluster";
+import { EventEmitter } from "../events";
+
+const _cluster = new EventEmitter() as Cluster;
+const _noop = () => {
+  return _cluster;
+};
+
+const on: typeof cluster.on = _noop;
```

Comment:
I tried to copy the existing pattern from `process`, but there isn't a global `cluster` object.